### PR TITLE
create-app: update changelog to put more emphasis on plugin-catalog-react migration

### PR DESCRIPTION
**Pull request by Rugvip**
It's a FAQ, so hoping to draw a bit more attention here. Will also update the release notes with the same message.

### CODE REVIEW

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4410-pqnhrx5cbpjivuhltk5zdflnpsq6ixr2.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601de0c1353bbb06bca18c2c

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4410-pqnhrx5cbpjivuhltk5zdflnpsq6ixr2.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601de0c1353bbb06bca18c2c

File: packages/create-app/CHANGELOG.md [hunk 1]
```diff
@@ -4,9 +4,9 @@
 
 ### Patch Changes
 
-- 019fe39a0: `@backstage/plugin-catalog` stopped exporting hooks and helpers for other
-  plugins. They are migrated to `@backstage/plugin-catalog-react`.
-  Change both your dependencies and imports to the new package.
+- 019fe39a0: **BREAKING CHANGE**: The `useEntity` hooks has been moved from `@backstage/plugin-catalog` to `@backstage/plugin-catalog-react`.
```
Comment (freben) [inline packages/create-app/CHANGELOG.md:7]:
```suggestion
- 019fe39a0: **BREAKING CHANGE**: The `useEntity` hook has been moved from `@backstage/plugin-catalog` to `@backstage/plugin-catalog-react`.
```

**Review by freben — APPROVED** [review]

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4410-pqnhrx5cbpjivuhltk5zdflnpsq6ixr2.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601de0c1353bbb06bca18c2c

**Pull request merged**